eq-author/public/index.html

```diff
@@ -1,20 +1,55 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <!-- Google Tag Manager -->
```

Comment:
Can we move this with the rest of the tracking code. They are still within the head.

Reply:
Yeah thats fine, the docs just said to put it as far up in the head as possible